### PR TITLE
dependencies: update to angular 11

### PR DESCRIPTION
**Pull request by jma**
Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

**Review by Garfield-fr — APPROVED** [review]

File: rero_ils/templates/rero_ils/professional.html [hunk 1]
```diff
@@ -26,7 +26,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <base href="/professional">
     <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
```
Comment (iGormilhit) [inline rero_ils/templates/rero_ils/professional.html:29]: ✌

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

**Comment by benerken** [comment]
I have installed rero-ils#1529 + rero-ils-ui#445 + ng-core#308
> cd ng-core ; npm ci ; npm run pack 
> cd ../rero-ils-ui ; npm ci ; npm -i ../ng-core/rero-ng-core-0.15.0.tgz ; npm run pack
> cd ../rero-ils ; poetry run bootstrap -t ../rero-ils-ui/build/rero-rero-ils-ui-0.8.0.tgz

- [ ] In the Professional interface Main screen (https://localhost:5000/professional/), the browser console displays
 some warnings like this:

image

- [ ] In the professional interface (Acquisition), when I try to add a new line to an order, I get the line editor but the console displays the message:

image

**Comment by jma** [comment]
> I have installed rero-ils#1529 + rero-ils-ui#445 + ng-core#308
> 
> > cd ng-core ; npm ci ; npm run pack
> > cd ../rero-ils-ui ; npm ci ; npm -i ../ng-core/rero-ng-core-0.15.0.tgz ; npm run pack
> > cd ../rero-ils ; poetry run bootstrap -t ../rero-ils-ui/build/rero-rero-ils-ui-0.8.0.tgz
> 
> _Tested with Firefox 81_
> 
> * [ ]  In the Professional interface Main screen (https://localhost:5000/professional/), the browser console displays
>   some warnings like this:
> 
> image
> 
> * [ ]  When you Log in or you log out, it works but the console displays this:
> 
> image

This will be solved in the next PR about webpack.

**Pull request closed without merging**